ihmc-perception/src/main/java/us/ihmc/perception/ffmpeg/FFmpegEncoder.java

```diff
@@ -127,19 +127,22 @@ public boolean encodeNextFrame(Consumer<AVPacket> packetConsumer)
       return error != AVERROR_EOF();
    }
 
+   public boolean flush(Consumer<AVPacket> packetConsumer)
```

Comment:
Turns out we should be flushing out the encoders once all the frames are sent (a few packets might've stayed in the encoder, waiting to be grouped together)